site/pages/offline/server-worker-diagram.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styles from './style.css';
+
+export default () => (
+  <svg className={styles.diagram} viewBox="0 0 268 402" version="1.1">
```

Comment:
You can use InlineSVG: https://github.com/redbadger/website-honestly/blob/master/site/components/case-study/index.js#L2